service/follower/record.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package follower
+
+import (
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/mempool/entity"
+)
+
+type Record struct {
+	Block       *flow.Block          `validate:"required"`
```

Comment:
Do these validation tags get used?

I saw validation mentioned elsewhere in the PR history, but didn't spot it now..

Reply:
It used to be validated but after the refactors that might have gone out the door 😄